openstates/mt/legislators.py

```diff
@@ -112,6 +118,7 @@ def scrape(self, chamber, term):
 
             office['phone'] = deets.get('phone')
             office['fax'] = deets.get('fax')
+            office['email'] = deets.get('email')
             legislator.add_office(**office)
```

Comment:
Is there any case in which they'll have phone/fax/email in `entry` but not in `deets`?